src/main/kotlin/com/grepp/nbe1_3_team04/vote/domain/Vote.kt

```diff
@@ -0,0 +1,80 @@
+package com.grepp.nbe1_3_team04.vote.domain
+
+import com.grepp.nbe1_3_team04.global.domain.BaseEntity
+import jakarta.persistence.*
+import jakarta.validation.constraints.NotNull
+import org.hibernate.annotations.SQLDelete
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+
+@Table(
+    indexes = [Index(name = "idx_vote_team_id", columnList = "teamId"), Index(
+        name = "idx_vote_member_id",
+        columnList = "memberId"
+    )]
+)
+@SQLDelete(sql = "UPDATE vote SET is_deleted = 'TRUE' WHERE vote_id = ?")
+@Entity
+class Vote private constructor(
+    val memberId: Long,
+    val teamId: Long,
+    title: String,
+    endAt: LocalDateTime
+) : BaseEntity() {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    val voteId: Long? = null
+
+    @Column(length = 50)
+    var title: @NotNull String? = null
+        protected set
+
+    var endAt: @NotNull LocalDateTime? = null
+        protected set
```

Comment:
이 부분이 초기화가 안돼서 null로 들어갔던거 같아요!
```suggestion
    @Column(length = 50)
    var title: @NotNull String = title
        protected set

    var endAt: @NotNull LocalDateTime = endAt
        protected set
```
이렇게 바꿔주시면 매개변수로 받은 값을 초기화시켜줄 수 있습니다!

Reply:
재경님이 알려주신대로 반영하였습니다 !